doc/source/getting_started/intro_tutorials/09_timeseries.rst

```diff
@@ -382,7 +382,7 @@ More details on the power of time series ``resampling`` is provided in the user
     <div class="d-flex flex-row gs-torefguide">
         <span class="badge badge-info">To user guide</span>
 
-A full overview on time series is given in the pages on :ref:`time series and date functionality <timeseries>`.
+A full overview on time series is given on the pages on :ref:`time series and date functionality <timeseries>`.
```

Comment:
i think this one is correct as "in"

Reply:
As far as I know, "on" is more correct here grammatically, but it sounds clunky, and we can avoid it. I would rephrase the whole thing like so:
For a full overview on time series, see :ref:`time series section <timeseries>` in the user guide.

Reply:
As far as I know, "on" is more correct here grammatically, but it sounds clunky, and we can avoid it. I would rephrase the whole thing like so:
For a full overview on time series, see :ref:`time series section <timeseries>` in the user guide.

Reply:
cc @WillAyd tie breaker?

Reply:
in is maybe more correct, but I don't think worth changing if its the only thing here. @AlexKirko if you want to reformat sentence separately would definitely be welcome